changelog/v1.18.0-rc3/platform-agnostic-lbephash.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+  - type: NON_USER_FACING
+    description: >-
+      Don't use xor or native endian-ness when generating hashes.
```

Comment:
can we update changelog (and PR description) to reflect that we are keeping xor ?